templates/base.html

```diff
@@ -8,10 +8,13 @@
   </head>
   <body>
     <div class="mainbar">
-      <h1><a href="/" class="nolinkstyle">PolarBear</a></h1>
+      <h1><a href="/" class="nolinkstyle" style = 'padding-left: 10pt'>PolarBear</a></h1>
       {% if user and user.is_authenticated %}
       <div class="baroption">
         <p style = 'padding-right: 10pt'>welcome, {{ user.username }}</p>
+        <a href="/user/authors/{{ user.author.id }}" class="buttonlink">Profile</a>
+        <a href="/" class="buttonlink">Home</a>
```

Comment:
this is unnecessary because the "PolarBear" title already links to the home page

Reply:
Most websites have a home button despite the title linking to home (Facebook, Twitter, Youtube, etc)
More intuitive place to click when wanting to go to the home page

Reply:
makse sense. approved!